### PR TITLE
[WIP] Fixing  navbar susper logo

### PR DESCRIPTION
**Pull request by subhahu123**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

-

- https://pr-1444-fossasia-susper.surge.sh

-

### CODE REVIEW

**Comment by subhahu123** [comment]
not ready yet..

**Comment by Pipe-Runner** [comment]
> not ready yet..

Put a [WIP] in the PR title if this is still a work in progress.

**Review by sakshee-19 — CHANGES_REQUESTED** [review]
Please also squash your commits.

File: src/app/about/about.component.css [hunk 1]
```diff
@@ -53,7 +53,12 @@ a:hover {
 }
 
 .navbar-brand {
-    padding-left: 100px;
+    padding: 0px;
+}
+
+#logo-link {
+    margin-left: 90%;
+	margin-top: 17%;
```
Comment (samagragupta) [inline src/app/about/about.component.css:61]: Make proper indentation.

**Review by samagragupta — COMMENTED** [review]
Please write the issue number.

**Comment by simsausaurabh** [comment]
Closing due to inactivity.
Please squash your commits properly and provide proper description of work done.

**Pull request closed without merging**